targets/targets.json

```diff
@@ -374,8 +374,13 @@
             "SERIAL",
             "SLEEP",
             "SPI",
-            "SPISLAVE"
+            "SPISLAVE",
+            "USTICKER"
         ],
+        "overrides": {
+            "tickless-from-us-ticker"   : true,
+            "boot-stack-size"           : "0x400"
```

Comment:
How did we come up with `0x400`?

Reply:
> How did we come up with 0x400?

It is from linker script file.
https://github.com/ARMmbed/mbed-os/blob/9243abb6be0cb43ffc40df675cbdd443a796ff66/targets/TARGET_NXP/TARGET_LPC11UXX/device/TOOLCHAIN_ARM_STD/TARGET_LPC11U35_401/LPC11U35.sct#L4

Reply:
Ah 0k. Thanks.